src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -36,6 +54,16 @@ public void GetValidators(ClientValidatorProviderContext context)
             {
                 throw new ArgumentNullException(nameof(context));
             }
+            IStringLocalizer stringLocalizer = null;
+            if (_options != null &&
+                _options.Value.DataAnnotationLocalizerProvider != null &&
+                _stringLocalizerFactory != null)
+            {
+                // This will pass first non-null type (either containerType or modelType) to delegate.
```

Comment:
This isn't a useful comment. - **WHY does it do this and why is that the right thing to do?**